payments-core/src/main/java/com/stripe/android/payments/core/authentication/IntentAuthenticatorRegistry.kt

```diff
@@ -1,11 +1,12 @@
 package com.stripe.android.payments.core.authentication
 
 import com.stripe.android.model.StripeIntent
+import com.stripe.android.payments.core.ActivityResultLauncherHost
 
 /**
  * Registry to map [StripeIntent] to the corresponding [IntentAuthenticator] to handle its next_action
  */
-internal fun interface IntentAuthenticatorRegistry {
+internal interface IntentAuthenticatorRegistry : ActivityResultLauncherHost {
```

Comment:
Why is a Registry a Host?

Reply:
Because later we might need the Registry to host `ActivityResultLauncher`s that are shared across the `IntentAuthenticator`s (e.g `paymentRelayLauncher` and `paymentBrowserAuthLauncher`).